### PR TITLE
Add sqlite-utils docset

### PR DESCRIPTION
**Pull request by Florents-Tselai**
This adds a docset for [sqlite-utils](https://github.com/simonw/sqlite-utils)

It also includes a `Makefile`, which further automates the process for most `sphinx`-powered documentation packages.

### CODE REVIEW

**Comment by Florents-Tselai** [comment]
Oops. Forgot to actually archive the docsets. 
Updating the pr...

**Comment by Florents-Tselai** [comment]
Looks like the CI doesn't properly handle multiple versions at once. Trying only the latest.

**Comment by Florents-Tselai** [comment]
If this is accepted, then I'd strongly suggest extracting the Makefile into a common directory and have it include-able in future docsets. ( I can open a separate PR for that)

File: docsets/sqlite-utils/docset.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "aliases": [
+  ],
+  "specific_versions": [
+    {
+      "author": {
+        "name": "Florents Tselai",
+        "link": "https://github.com/Florents-Tselai"
+      },
+      "version": "3.35.2",
+      "archive": "versions/3.35.2/sqlite-utils.tgz"
```
Comment (Kapeli) [inline docsets/sqlite-utils/docset.json:18]: This archive needs to be added as well.

**Comment by Kapeli** [comment]
> If this is accepted, then I'd strongly suggest extracting the Makefile into a common directory and have it include-able in future docsets. ( I can open a separate PR for that)

Dash is now able to generate docsets for Sphinx docs on its own now, starting with version 7.2.0.

File: docsets/sqlite-utils/docset.json (hunk 1 above)
Comment (Florents-Tselai) [reply docsets/sqlite-utils/docset.json:18]: Added.

**Comment by Kapeli** [comment]
Thank you!

**Pull request merged**